pkg/shared/findingkey/malware.go

```diff
@@ -25,19 +25,17 @@ import (
 type MalwareKey struct {
 	MalwareName string
 	MalwareType string
-	Path        string
 	RuleName    string
 }
 
 func (k MalwareKey) String() string {
-	return fmt.Sprintf("%s.%s.%s.%s", k.MalwareName, k.MalwareType, k.Path, k.RuleName)
+	return fmt.Sprintf("%s.%s.%s", k.MalwareName, k.MalwareType, k.RuleName)
 }
 
 func GenerateMalwareKey(info models.MalwareFindingInfo) MalwareKey {
 	return MalwareKey{
 		MalwareName: utils.ValueOrZero(info.MalwareName),
 		MalwareType: utils.ValueOrZero(info.MalwareType),
-		Path:        utils.ValueOrZero(info.Path),
```

Comment:
For a finding the same malware found on a different path should be considered a unique finding. But in the widget when we're just looking for "affected assets" we shouldn't take that path into account.